sra_to_salmon.pbs

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -l
+
+################################################################################
+#Script Name: sra_to_salmon.pbs
+#Description: Submission script for sra_to_salmon job arrays
+#Args       : None 
+#Author     : Georgia Doing
+#Email      : Georgia.Doing.GR@Dartmouth.edu
+################################################################################
+
+#PBS -q largeq
+# #PBS -A NCCC
+#PBS -N byx
+#PBS -l walltime=00:30:00
+#PBS -l nodes=1:ppn=1
+#PBS -m bea
+#PBS -M Georgia.Doing.GR@Dartmouth.edu
+#PBS -t 1-3013%20
+#PBS -j oe
+#PBS -e /dartfs-hpc/scratch/f002bx6/salmon/sra_comp/oe
+#PBS -o /dartfs-hpc/scratch/f002bx6/salmon/sra_comp/oe
+
+'''
+This is a job array submission script that passes array
+indices and a directory to a bash script which initiates
+the processing of the experiment-specific file (SRX or ERX)
+in the specified directory at the specified index.
+
+This level of parallelization, at the expeirment level,
+allows for each job to download and process together (in a
+single salmon quant call) the sequence
+data associated with a single experiement accession, 
+which for this data is 1-10 fastq files and 8KB - 18GB. Each
+job is given 1 processor and 30 min, 3013 jobs are submitted
+and run 20 at a time.
+
+To the best of my understanding, SRA experiment accessions
```

Comment:
Just curious, what did you mean when you said that NCBI "experiments" were equivalent to what you would normally call a "sample"? I guess I think of an experiment have a group of samples that have a shared background/hypothesis. Which is what i thought NCBI experiments were...but its been a while.

Reply:
From what I can tell based on sample names, NCBI SRA projects are what people normally think of as an experiment: they are groups of samples. NCBI calls each sample in a project an SRA 'experiment'. So experiments can consist of data from multiple runs but they are from the same mRNA library prep..I should put some examples in slides for our group meeting just so we can make sure I am correct about this, it has been a bear to figure out, haha